src/main/java/org/opentripplanner/routing/impl/OrcaFareServiceImpl.java

```diff
@@ -483,7 +484,11 @@ public boolean populateFare(Fare fare,
                 // If the new fare is more than the current ORCA amount, the transfer is extended.
                 if (legFare > orcaFareDiscount) {
                     freeTransferStartTime = ride.startTime;
+                    addFareComponent(ride, fareType, currency, legFare - orcaFareDiscount, true);
```

Comment:
Could you clarify the following please?

In the results, the first transit leg of a fare list seems to show `isTransfer = true` while showing the initial price paid. Intuitively, the first transit leg would have `isTransfer = false` (as for any subsequent transit leg where the fare "resets", which the code correctly assigns `isTransfer = false`).

```yaml
...
electronicYouth:
  0:
    isTransfer: true
    legIndex: 1
    price:
      cents: 150
      ...
```

Reply:
Hm, you're right. This is an error. Should be a simple fix.